api/v1alpha1/operator_types.go

```diff
@@ -45,6 +45,18 @@ type OperatorStatus struct {
 	// then it means the Operator has either not been installed yet or is failing to install.
 	// +optional
 	ActiveBundleDeployment ActiveBundleDeployment `json:"activeBundleDeployment,omitempty"`
+
+	// sourceInfo is the reference to the previously sourced catalog data that's being
+	// installed by this Operator resource.
+	SourceInfo SourceInfo `json:"sourceInfo,omitempty"`
```

Comment:
This might introduce some potential coupling in the internal implementations as my current plan is to leverage this information if it's already present to avoid cross-catalog sourcing after an operator has been installed.